### ISSUE TITLE
GcsStorageService : Failed to load notifications

### ISSUE THREAD

**Issue by sdpoueme** [spinnaker/spinnaker#5698]
### Issue Summary:
Front50 is returning an exception which seems to be related to #3144  

### Cloud Provider(s):

OpenStack

### Environment:

Front50

### Feature Area:

Performance

### Description:

There is no visible on the existing pipelines, they run fine and can send notifications. 
There is a correlation between that error and the usage of Spinnaker. 
![image](https://user-images.githubusercontent.com/10876571/79813018-80340c00-8348-11ea-9787-33577116e6e9.png)

The issue happens a lot more when there is more activity on the platform. 

### Steps to Reproduce:
No specific steps:
-  Put some load on spinnaker by executing pipelines for different applications at the same time. 
-  Let the system rest. 
- Compare ELK logs between both states

### Additional Details:

`log:2020-04-21 00:45:03.461 ERROR 1 --- [ificationDAO-17] c.n.s.front50.model.GcsStorageService : Failed to load notifications __global: 404 Not Found @timestamp:April 20th 2020, 20:45:03.462 stream:stdout time:April 20th 2020, 20:45:03.462 kubernetes.pod_name:spin-front50-5d48897fbc-bn59d`

---

**Comment by ezimanyi** [comment]
@sdpoueme : Thanks for reporting this!  I've looked into it and this is just a case of an unnecessary log statement.  In cases where there aren't application-wide notifications enabled, we expect to get back a 404 when looking for them, and do actually properly catch (and ignore) the exception.  But we do log that the exception occurred, which is not helpful given that there's not actually any error here.

In the above-linked PR, I've made a change to remove the log statement as it's not necessary.
